### PR TITLE
[TH2-1598] Add IN, LIKE, MORE, LESS filter operation

### PR DESCRIPTION
**Pull request by Sergey-Sitnikov-Exactpro**

### CODE REVIEW

File: src/main/proto/th2_grpc_common/common.proto [hunk 1]
```diff
@@ -154,6 +154,10 @@ enum FilterOperation {
   NOT_EQUAL = 1;
   EMPTY = 2;
   NOT_EMPTY = 3;
+  IN = 4;
+  LIKE = 5;
+  MORE = 6;
+  LESS = 7;
```
Comment (OptimumOpium) [inline src/main/proto/th2_grpc_common/common.proto:160]: Please, add negative versions of those operations

File: src/main/proto/th2_grpc_common/common.proto [hunk 2]
```diff
@@ -163,13 +167,18 @@ message ValueFilter {
     string simple_filter = 3;
     MessageFilter message_filter = 4;
     ListValueFilter list_filter = 5;
+    ComplexFilter  complex_filter= 6;
```
Comment (OptimumOpium) [inline src/main/proto/th2_grpc_common/common.proto:170]: I think we should call it SimpleList or something like that. It sounds more correctly in my oppinion

**Review by OptimumOpium — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -42,17 +42,17 @@ If you wish to manually create and publish a package for Python:
 
 + Added `IN` and 'NOT_IN' filter operation, which filter a field by value from list
 + Added `LIKE` and 'NOT_LIKE' filter operation, which filter a field by regEx expression
-+ Added `MORE`, `LESS`, 'NOT_MORE', 'NOT_LESS' filter operations, which filter a field by comparing numeric values
++ Added `MORE`, `LESS`, 'NOT_MORE', 'NOT_LESS' filter operations, which filter a field by comparing values
```
[the remark this answers is not in the document]
Comment (Sergey-Sitnikov-Exactpro) [reply README.md:45]: Should I specify types like java class name (e.g. LocalDate) or 
Added `MORE`, `LESS`, `NOT_MORE`, `NOT_LESS` filter operations, which filter a field by comparing values(`numeric` types and `date/time` in ISO format) is enough?

[the remark this answers is not in the document]
Comment (OptimumOpium) [reply README.md:45]: I guess the second option is the best one

**Review by OptimumOpium — APPROVED** [review]

**Pull request merged**